cypress/integration/timeline_email.js

```diff
@@ -0,0 +1,71 @@
+context('Timeline Email', () => {
+	before(() => {
+		cy.visit('/login');
+		cy.login();
+		cy.visit('/app/todo');
+	});
+
+	it('Adding new ToDo, adding email and verifying timeline content for email attachment, deleting attachment and ToDo', () => {
+		//Adding new ToDo
+		cy.click_listview_primary_button('Add ToDo');
+		cy.get('.custom-actions > .btn').trigger('click', {delay: 500});
+		cy.get('.row > .section-body > .form-column > form > .frappe-control > .form-group > .control-input-wrapper > .control-input > .ql-container > .ql-editor').eq(0).type('Test ToDo', {force: true});
+		cy.wait(500);
+		//cy.click_listview_primary_button('Save');
+		cy.get('.primary-action').contains('Save').click({force: true});
+		cy.wait(700);
+		cy.visit('/app/todo');
+		cy.get('.list-row > .level-left > .list-subject > .level-item.ellipsis > .ellipsis').eq(0).click();
+
+		//Creating a new email
+		cy.get('.timeline-actions > .btn').click();
+		cy.fill_field('recipients', 'test@example.com', 'MultiSelect');
+		cy.get('.modal.show > .modal-dialog > .modal-content > .modal-body > :nth-child(1) > .form-layout > .form-page > :nth-child(3) > .section-body > .form-column > form > [data-fieldtype="Text Editor"] > .form-group > .control-input-wrapper > .control-input > .ql-container > .ql-editor').type('Test Mail');
```

Comment:
The email test is failing for new PRs. 

The problem is deeper than a particular test. A UI test that relies on a 270 character long selector is bound to break with even minor changes in UI that don't break any functionality.

Consider using https://testing-library.com/ ? 

@surajshetty3416 @Komal-Saraf0609